src/core/CoreTransaction.jl

```diff
@@ -58,13 +58,28 @@ end
 
 function query(transaction::T, request::R, batch::Bool) where {T<:AbstractCoreTransaction, R<:Proto.ProtoType}
     !is_open(transaction) && throw(TypeDBClientException(CLIENT_TRANSACTION_CLOSED))
-    result = single_request(transaction.bidirectional_stream, request, batch)
+    result = []
```

Comment:
Not type stable. Same comment for `stream` function below.

Reply:
You are right and now it is changed.